src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/ReflectionJsonTypeInfoOfT.cs

```diff
@@ -37,11 +32,17 @@ internal ReflectionJsonTypeInfo(JsonConverter converter, JsonSerializerOptions o
             converter.ConfigureJsonTypeInfoUsingReflection(this, options);
         }
 
-        [RequiresUnreferencedCode(JsonSerializer.SerializationUnreferencedCodeMessage)]
```

Comment:
is this change relevant to actual fix?

Reply:
Because the logic has been moved to `LateAddProperties` which is a virtual method, linker/AOT annotations must match that of the base method (which doesn't have any). In this case it safe to suppress warnings since it's an instance method on a type with annotated constructors.